src/sources/aws_s3/sqs.rs

```diff
@@ -672,30 +678,30 @@ impl Serialize for S3EventName {
     where
         S: Serializer,
     {
-        serializer.serialize_str(&format!("{}:{}", self.name, self.kind))
+        serializer.serialize_str(&format!("{}:{}", self.kind, self.name))
```

Comment:
This one really confused me for a bit :sweat_smile:

Reply:
Ouch, yeah, I bet.